radicle/src/cob/patch.rs

```diff
@@ -394,10 +394,34 @@ impl Revision {
         self.description.get()
     }
 
+    pub fn author(&self) -> &Author {
+        &self.author
+    }
+
+    pub fn base(&self) -> &git::Oid {
+        &self.base
+    }
+
+    pub fn oid(&self) -> &git::Oid {
```

Comment:
We should call this `head` I think, would be more obvious what oid it refers to.